oci.go

```diff
@@ -22,13 +22,18 @@ import (
 const (
 	ociConfigFile     string      = "config.json"
 	ociConfigFileMode os.FileMode = 0444
+	ociConfigBasePath string      = "/run/libcontainer"
 )
 
-// writeSpecToFile writes the container's OCI spec to "dirname(spec.Root.Path)/config.json"
-// This effectively makes the parent directory a valid OCI bundle.
-func writeSpecToFile(spec *specs.Spec) error {
-	bundlePath := filepath.Dir(spec.Root.Path)
-	configPath := filepath.Join(bundlePath, ociConfigFile)
+// writeSpecToFile writes the container's OCI spec to "/run/libcontainer/<container-id>/config.json"
```

Comment:
jfyi - this directory already exist in the guest

```
# ls -lR /run/libcontainer/
/run/libcontainer/:
total 0
drwx------    3 root     root            60 Jun 22 17:40 078a6955095206cb6cf326c82a1dc6d7f5329edab206774f71a668d1b95ea172

/run/libcontainer/078a6955095206cb6cf326c82a1dc6d7f5329edab206774f71a668d1b95ea172:
total 0
drwx--x--x    2 root     root            60 Jun 22 17:40 078a6955095206cb6cf326c82a1dc6d7f5329edab206774f71a668d1b95ea172

/run/libcontainer/078a6955095206cb6cf326c82a1dc6d7f5329edab206774f71a668d1b95ea172/078a6955095206cb6cf326c82a1dc6d7f5329edab206774f71a668d1b95ea172:
total 12
-rw-r--r--    1 root     root         10033 Jun 22 17:40 state.json
```

Reply:
Yes @devimc. The root `/run/libcontainer` exists, however the container sub-dir doesn't exist when writeSpecToFile gets called.

Reply:
@bpradipt Do we already have directory /run/libcontainer/<pod-id>/<container-id> ?
If not maybe just use that ?

Reply:
@amshinde when writeSpecToFile is called, we only have `/run/{kata-containers, sandbox-ns}`